improver/calibration/rainforest_calibration.py

```diff
@@ -214,6 +215,9 @@ def __init__(self, model_config_dict: dict, threads: int = 1):
                 np.array([*sorted_model_config_dict[self.lead_times[0]].keys()])
             )
         else:
+            warnings.warn(
+                "Model config does not contain lead time keys; calibration will not work"
+            )
```

Comment:
Thanks for adding this. I think you could go one step further and say that calibration process will fail.

Reply:
If you change this, you'll need to update the associated test.